vkcs/monitoring/resource_cloud_monitoring_test.go

```diff
@@ -0,0 +1,63 @@
+package monitoring_test
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/vk-cs/terraform-provider-vkcs/vkcs/internal/acctest"
+)
+
+func TestAccCloudMonitoring_basic(t *testing.T) {
+	resourceName := "vkcs_cloud_monitoring.basic"
+	resource.Test(t, resource.TestCase{
+		PreCheck:                 func() { acctest.AccTestPreCheck(t) },
+		ProtoV6ProviderFactories: acctest.AccTestProtoV6ProviderFactories,
+		Steps: []resource.TestStep{
+			{
+				Config: acctest.AccTestRenderConfig(testAccCloudMonitoringBase, map[string]string{"BaseImageDataSource": testAccImageUbuntuDataSource}),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttrSet(resourceName, "service_user_id"),
+					resource.TestCheckResourceAttrSet(resourceName, "script"),
+				),
+			},
+			{
```

Comment:
Basic test should only have one step. This kind of tests conventionally means that a test checks the minimum possible valid configuration. If you want to add test for an update, add an extra one named correspondingly.

Reply:
add second test

Reply:
I don't see it. Everything remains the same.

Reply:
I messed up the tests, now everything is fixed.